config/api.yml

```diff
@@ -666,10 +666,12 @@
   :cloud_volumes:
     :description: Cloud Volumes
     :identifier: cloud_volume
-    :options:
+    options:
```

Comment:
we lost the leading :

Reply:
Thx, fixed. The yarm parser did not mind, but we should be consistent.